src/components/site/tableOfContents/index.jsx

```diff
@@ -0,0 +1,112 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React from 'react';
+import reactn from 'reactn';
+import { Link, graphql, StaticQuery } from 'gatsby';
+import PropTypes from 'prop-types';
+import { Drawer } from 'react-md';
+import styles from './table-of-contents.module.scss';
+
+// import Footer from '../components/site/footer';
+const TO_PREFIX = '/';
+const routes = [
+  {
+    label: 'home',
+    to: `${TO_PREFIX}`,
+    icon: 'home',
+    exact: 'true',
+    primaryText: 'Home',
+    active: false,
+  },
+  {
+    label: 'style-guide',
+    to: `${TO_PREFIX}StyleGuide`,
+    icon: 'style',
+    primaryText: 'Style Guide',
+    active: false,
+  },
+  { divider: true },
+  {
+    primaryText: 'Table of Contents',
+    subheader: true,
+  },
+];
```

Comment:
Unless there's a reason this has to live outside the class, let's move these constants into the constructor like:
```
this.To_PREFIX = ....
this.routes = ....
```
and of course then update the use of these variables accordingly.

Reply:
moved into class